Week1/QA_PREP_EXERCISE.md

```diff
@@ -0,0 +1,17 @@
+As a preparation step for the upcoming Q&A, you will work through the following exercise. The expected result is a list of SQL `CREATE TABLE` statements that represent the tables you're being asked to design. We suggest you first think of the design of the tables (names, columns, types) and then proceed to write the SQL statements, that way you are thinking about the problem you're trying to solve (creating a database for a website), rather than the implementation first (creating a set of SQL statements).
+
+### Option 1
+
+Design the tables for a database that represents a website with food recipes. You don't need to write any specific queries at the moment, but the design should consider the need for some future queries.
+
+- Each recipe should have a name, a category, a list of ingredients, and a list of steps to follow to complete the recipe. Many recipes might share the same ingredients or the same list of steps (For example, cook pasta according to the package instructions).
```

Comment:
> For example, cook pasta according to the package instructions

This example is not clear to me. Could you please elaborate ?

Reply:
Yes, this might be controversial. What I'm trying to do here is see if students will come up with an entity "recipe_step", which would be a somewhat awkward one given the nature of recipes. This is mostly to later illustrate that an entity relationship in this case is not the best idea, and hence using a document db can be better suited.